crates/op-rbuilder/README.md

```diff
@@ -14,7 +14,7 @@ To run op-rbuilder with the op-stack, you need:
 To run the op-rbuilder, run:
 
 ```bash
-cargo run -p op-rbuilder --bin op-rbuilder --features optimism -- node \
+cargo run -p op-rbuilder --bin op-rbuilder --features flashblocks -- node \
```

Comment:
flashblocks is not default, would be better to have a separate flashblocks sections to enable it here

Reply:
Sounds great, I'll update this PR.